images/linux/scripts/base/apt-mock.sh

```diff
@@ -13,11 +13,10 @@ i=1
 while [ \$i -le 30 ];do
   err=\$(mktemp)
   $real_tool "\$@" 2>\$err
-  result=\$?
-  cat \$err >&2
 
-  # no errors, continue
-  test \$result -eq 0 && break
+  # no errors, break the loop and continue normal flow
+  test -f \$err || break
```

Comment:
Is it safe to not look at $result?
The following cases are possible? :
- $result == 0 && $err != ""
- $result != 0 && $err == ""

cc: @al-cheb @miketimofeev

Reply:
if even $result != 0 && $err == "" we do not handle the case (all grep-s ignored) anyway